static/css/page-signup.less

```
@@ -0,0 +1,156 @@
+/**
+ * Stylesheet for signup and (soon) login forms
+
+ * Used in templates:
+ * - openlibrary/templates/account/create.html
+ *
+ * Used on pages:
+ * - /account/create
+ */
+
+@import (less) "less/index.less";
+
+// Import all base styles
+@import (less) "base/index.less";
+
+// Import colors
+@import (less) "less/colors.less";
+
+// layout rules
+@import (less) "layout/index.less";
+@import (less) "page-user.less";
+
+// Components (modules) in order of appearance on page
+@import (less) "components/header.less";
+@import (less) "components/form.olform.less";
+@import (less) "components/footer.less";
+@import (less) "components/language.less";
+
+.ol-page-signup {
+  display: flex;
+  flex-direction: row;
+
+  .ol-signup-hero > *, .ol-signup-form > * {
+    width: 70%;
+  }
+
+  .ol-signup-hero {
+    width: 55%;
+    background-color: @grey-fafafa;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    gap: 10px;
+    border-radius: 5px 0 0 5px;
+
+    h1 {
+      color: @black;
+    }
+
+    .ol-signup-form__subtitle {
+      font-size: 18px;
+    }
+
+    li {
+      margin-bottom: 18px;
+    }
+
+    @media only screen and (max-width: @width-breakpoint-tablet) {
+      display: none;
+    }
+  }
+
+  .ol-signup-form__link {
+    color: @link-blue;
+    text-decoration: none;
+    border-bottom: 1px solid @link-blue;
+  }
+
+  .ol-signup-form__link--muted, .ol-signup-form__newsletter a {
+    color: inherit;
+  }
+
```

We might want to flip this around a bit, since only the login one is unique. The others should already color: inherit. And we can use the default text-decoration: underline instead of border-bottom even though it looks a bit different from the mocks 😁 (I also just updated the mocks not to do that odd line under the text thing and just underline the text  :P)

```suggestion
  .ol-signup-form__link--login {
    color: @link-blue;
  }
```